### PR TITLE
bugfix/pattern city

### PR DESCRIPTION
**Pull request by jjeroch**
missing scenarios to cover: 
- St. Catharines
- Val-d'Or
- Presqu'ile

### CODE REVIEW

**Comment by jjeroch** [comment]
@nidhigarg-bmw  I updated the pattern to more fit for required city input validations.
Do you have an idea how we get additionally 

- St. Catharines
- Val-d'Or
- Presqu'ile

also covered inside the pattern.

**Review by oyo — APPROVED** [review]

File: src/helpers/utils.ts [hunk 1]
```diff
@@ -219,7 +219,7 @@ export const PATTERNS = {
     /^[a-zA-ZÀ-ÿ\d][a-zA-ZÀ-ÿ\d !#'$@&%()*+,\-_./:;=<>?[\]\\^]{2,60}$/,
   streetHouseNumberPattern: /^[a-zÀ-ÿA-Z\d][a-zA-ZÀ-ÿ\d -]{2,60}$/,
   postalCodePattern: /^(?=[a-zA-Z\d]*[-\s]?[a-zA-Z\d]*$)[a-zA-Z\d-\s]{2,10}$/,
-  cityPattern: /^[A-Za-zÀ-ÿ]{3,20}$/,
+  cityPattern: /^[A-Za-zÀ-ÿ]{2,20}+(?:[\s-][a-zA-Z]{2,20}+)*$/,
```
Comment (oyo) [inline src/helpers/utils.ts:222]: this doesn't look correct. You can test your patterns in the browser easily. in the dev tools console at the prompt write:

`/^[A-Za-zÀ-ÿ]{2,20}+(?:[\s-][a-zA-Z]{2,20}+)*$/.test('Munich')`

or any other city you like to check

**Review by oyo — CHANGES_REQUESTED** [review]

**Comment by jjeroch** [comment]
@oyo  we spoke:
Test values - all should be valid :)

- Toronto
- St. Catharines
- San Fransisco
- Val-d'Or
- Presqu'ile
- Niagara on the Lake
- Niagara-on-the-Lake
- München
- Villes du Québec
- Provence-Alpes-Côte d'Azur
- Île-de-France
- Kópavogur
- Garðabær
- Sauðárkrókur
- Þorlákshöfn
- Munich

**Review by oyo — APPROVED** [review]
sehr schön ;)

**Pull request merged**